HTTPShortcuts/app/src/main/kotlin/ch/rmy/android/http_shortcuts/variables/types/DateType.kt

```diff
@@ -50,7 +50,7 @@ constructor(
         const val KEY_FORMAT = "format"
         private const val DEFAULT_FORMAT = "yyyy-MM-dd"
 
-        internal val DATE_FORMAT: DateTimeFormatter = DateTimeFormatter.ofPattern(DEFAULT_FORMAT, Locale.US)
+        internal val DATE_FORMAT: DateTimeFormatter = DateTimeFormatter.ofPattern(DEFAULT_FORMAT, Locale.getDefault())
```

Comment:
Here the locale is intentionally hardcoded, as this is the format that's use for storing and restoring the date value.  To ensure that the date can be consistently stored and restored, even when the locale changes, it must always use the same locale.